iana_matroska_ids.md

```diff
@@ -9,11 +9,22 @@ To register a new Element ID in this registry, one needs an Element ID,
 a Change Controller (IETF or email of registrant) and
 an optional Reference to a document describing the Element ID.
 
-Element IDs are described in Section 5 of [@!RFC8794].  Element IDs are encoded
+Element IDs are encoded
 using the VINT mechanism described in Section 4 of [@!RFC8794] and can be between
 one and five octets long. Five-octet-long Element IDs are possible
 only if declared in the EBML header.
 
+Element IDs are described in Section 5 of [@!RFC8794]. 
+However IANA is requested to revise the registration procedures for that registry, based on Errata 7189, as follows:
```

Comment:
This is done in IANA, so we don't need this phrase anymore.
https://www.iana.org/assignments/ebml